Mono.Cecil/TypeDefinition.cs

```diff
@@ -284,6 +287,19 @@ void ResolveLayout ()
 			get { return generic_parameters ?? (this.GetGenericParameters (ref generic_parameters, Module)); }
 		}
 
+		public bool HasCustomDebugInformations {
+			get {
+				if (custom_infos != null)
+					return custom_infos.Count > 0;
+
+				return this.GetHasCustomDebugInformations (ref custom_infos, Module);
```

Comment:
Is there a reason to do this without also assigning the rows read to `custom_infos`? I'd imagine any code calling this would go on to look at `CustomDebugInformation`. Would it make sense to use `this.GetCustomDebugInfomations` and remove `GetHasCustomDebugInfomations`?